### PR TITLE
Update SetDnsServerDiagnostics.md EventLogLevel Description

### PR DESCRIPTION
**Pull request by X-Guardian**
EventLogLevel is of type `UInt32`, so won't accept the string values that are currently specified.

Updated the description to include the correct numerical values mapped to the options available on the `Event Logging` tab of the DNS Manager interface.

### CODE REVIEW

**Comment by e0i** [comment]
@X-Guardian Thank you for your contribution. Please target the branch of this PR to `master` to proceed with the review.

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/dnsserver/Set-DnsServerDiagnostics.md [hunk 1]
```diff
@@ -419,7 +419,15 @@ Accept wildcard characters: False
 
 ### -EventLogLevel
 Specifies an event log level.
-Valid values are Warning, Error, and None.
+Valid values are:
+
+0: No Events
```
Comment (dariomws306) [inline docset/windows/dnsserver/Set-DnsServerDiagnostics.md]:
```suggestion
- 0: No Events
```

File: docset/windows/dnsserver/Set-DnsServerDiagnostics.md [hunk 2]
```diff
@@ -419,7 +419,15 @@ Accept wildcard characters: False
...
+
+1: Errors Only
```
Comment (dariomws306) [inline docset/windows/dnsserver/Set-DnsServerDiagnostics.md]:
```suggestion
- 1: Errors Only
```

File: docset/windows/dnsserver/Set-DnsServerDiagnostics.md [hunk 3]
```diff
@@ -419,7 +419,15 @@ Accept wildcard characters: False
...
+
+2: Errors and warnings
```
Comment (dariomws306) [inline docset/windows/dnsserver/Set-DnsServerDiagnostics.md]:
```suggestion
- 2: Errors and warnings
```

File: docset/windows/dnsserver/Set-DnsServerDiagnostics.md [hunk 4]
```diff
@@ -419,7 +419,15 @@ Accept wildcard characters: False
...
+
+7: All Events
```
Comment (dariomws306) [inline docset/windows/dnsserver/Set-DnsServerDiagnostics.md]:
```suggestion
- 3-7: All Events
```

**Comment by get-itips** [comment]
I am ok once the suggestions are applied

File: docset/windows/dnsserver/Set-DnsServerDiagnostics.md (hunk 1 above)
Comment (X-Guardian) [reply docset/windows/dnsserver/Set-DnsServerDiagnostics.md]: I didn't use a markdown list on purpose, to match the style of the rest of the document, which contains 'lists' of options on other parameters, but does not user markdown lists. i.e. `DebugLogging` and `OperationalLogLevel1`. What would you like me to do, make the list styles inconsistent or 'apply markdown lists to the other lists too?

File: docset/windows/dnsserver/Set-DnsServerDiagnostics.md (hunk 4 above)
Comment (dariomws306) [reply docset/windows/dnsserver/Set-DnsServerDiagnostics.md]: This and if you can apply the same update for 2012 and 2012 R2 versions are more than enough for me. Thanks!

File: docset/windows/dnsserver/Set-DnsServerDiagnostics.md (hunk 1 above)
Comment (dariomws306) [reply docset/windows/dnsserver/Set-DnsServerDiagnostics.md]: It's true, I'm agree with you. Maybe it looks better but you choose.

Comment (dariomws306) [reply docset/windows/dnsserver/Set-DnsServerDiagnostics.md]: Agree with you.

File: docset/windows/dnsserver/Set-DnsServerDiagnostics.md (hunk 4 above)
Comment (dariomws306) [reply docset/windows/dnsserver/Set-DnsServerDiagnostics.md]: This update and apply the same for 2012 and 2012 R2 versions are more than enough for me. Thanks!

**Comment by tseward** [comment]
Agreed with @dariomws306's suggestion.

**Comment by e0i** [comment]
@dariomws306 Any additional input on revised content?

**Pull request merged**